src/trans-netlist/netlist.cpp

```diff
@@ -65,6 +65,17 @@ void netlistt::print(std::ostream &out) const
 
   out << '\n';
 
+  out << "State invariant constraints: " << '\n';
+
+  for(unsigned i = 0; i < constraints.size(); i++)
```

Comment:
`for(const auto &constraint : constraints)` unless I'm missing something (or at least `std::size_t` instead of `unsigned`).